cogs/memes.py

```diff
@@ -23,16 +23,32 @@
 
 # Other utilities
 import random
+from .utils.auto_incorrect import auto_incorrect
 
 
 class Memes():
     def __init__(self, bot):
         self.bot = bot
         self.mix_command_logger = open('mix.log', 'a')
+        self.bac_command_logger = open('bac.log', 'a')
```

Comment:
We should give the bot its own logger so that everything can be logged in the same file. Other than that it's all good